src/SourceBuild/content/test/Microsoft.DotNet.Installer.Tests/LinuxInstallerTests.cs

```diff
@@ -0,0 +1,373 @@
+        AddPackage(packageList, "aspnetcore-runtime-", packageType);
+        AddPackage(packageList, "aspnetcore-targeting-pack-", packageType);
+        AddPackage(packageList, "dotnet-apphost-pack-", packageType);
+        if (Config.Architecture == "x64")
+        {
+            // netstandard package exists for x64 only
+            AddPackage(packageList, "netstandard-targeting-pack-", packageType);
+        }
+        AddPackage(packageList, "dotnet-sdk-", packageType);
+
+        string dockerfile = GenerateDockerfile(packageList, baseImage, packageType);
+
+        string tag = $"test-{Path.GetRandomFileName()}";
+        string output = "";
+        bool buildCompleted = false;
+
+        try
+        {
+            // Build docker image and run the tests
+            _dockerHelper.Build(tag, dockerfile: dockerfile, contextDir: _contextDir);
+            buildCompleted = true;
+            output = _dockerHelper.Run(tag, tag);
+
+            int testResultsSummaryIndex = output.IndexOf("Tests run: ");
+            if (testResultsSummaryIndex >= 0)
+            {
+                string testResultsSummary = output[testResultsSummaryIndex..];
+                Assert.False(AnyTestFailures(testResultsSummary), testResultsSummary);
+            }
+            else
+            {
+                Assert.Fail("Test summary not found");
+            }
+        }
+        catch (Exception e)
+        {
+            if (string.IsNullOrEmpty(output))
+            {
+                output = e.Message;
+            }
+            Assert.Fail($"{(buildCompleted ? "Build" : "Test")} failed: {output}");
+        }
+        finally
+        {
+            if (!Config.KeepDockerImages)
+            {
+                _dockerHelper.DeleteImage(tag);
+            }
+        }
+    }
+
+    private string GenerateDockerfile(List<string> packageList, string baseImage, PackageType packageType)
+    {
+        StringBuilder sb = new();
+        sb.AppendLine("FROM " + baseImage);
+        sb.AppendLine("");
+        sb.AppendLine("# Copy NuGet.config");
+        sb.AppendLine($"COPY NuGet.config .");
+
+        sb.AppendLine("");
+        sb.AppendLine("# Copy scenario-tests content");
+        sb.AppendLine($"COPY scenario-tests scenario-tests");
+
+        sb.AppendLine("");
+        sb.AppendLine("# Copy nuget packages");
+        sb.AppendLine($"COPY packages packages");
+
+        sb.AppendLine("");
+        sb.AppendLine("# Copy installer packages");
+        foreach (string package in packageList)
+        {
+            sb.AppendLine($"COPY {package} {package}");
+        }
+        sb.AppendLine("");
+        sb.AppendLine("# Install the installer packages and Microsoft.DotNet.ScenarioTests.SdkTemplateTests tool");
+        sb.Append("RUN");
+
+        // TODO: remove --force-all when aspnet package versioning issue have been resolved - https://github.com/dotnet/source-build/issues/4895
```

Comment:
It looks like this has been resolved.  Can the workaround be removed now?

Reply:
Yes, this should be in VMR now - will confirm and update the code.

Reply:
We still need this workaround due to Debain DEPs package that requires `libgssapi-krb5-2`. The package is missing on `mcr.microsoft.com/dotnet/nightly/runtime-deps:10.0-preview-trixie-slim`

https://github.com/dotnet/dotnet-docker/issues/6271